src/EVM.hs

```diff
@@ -834,13 +834,18 @@ exec1 = do
             xValue:xOffset:xSize:xs ->
               accessMemoryRange xOffset xSize $ do
                 availableGas <- use (#state % #gas)
-                let
-                  (cost, gas') = costOfCreate fees availableGas xSize False
-                newAddr <- createAddress self this.nonce
+                let (cost, gas') = costOfCreate fees availableGas xSize False
+
+                -- handle `prank`
+                let from' = fromMaybe self vm.config.overrideCaller
+                resetCaller <- use (#config % #resetCaller)
+                when (resetCaller) $ assign (#config % #overrideCaller) Nothing
```

Comment:
Can you explain for me what these two lines do?

Reply:
These are no longer there. But they reset the caller after `prank`. Only when `startPrank` should the caller remain the pranked caller. I am closing because this has been fixed as part of the #240 issue. Let me know if you have any questions, I'd be happy to answer. It's perhaps worthwhile reading the associated PR that closed that issue.